content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started/getting-started.md

```diff
@@ -29,7 +29,9 @@ Now that you have played with Alvik and have seen it moving, it is time to know
 
 ### Let's Start Coding Alvik
 
-Alvik is intended to be programmed with MicroPyton. We recommend you to install the [Arduino Lab for MicroPython](https://labs.arduino.cc/en/labs/micropython) editor. Follow the instructions next to the download link to install it and open the IDE
+Alvik is intended to be programmed with MicroPyton. We recommend you to install the [Arduino Lab for MicroPython](https://labs.arduino.cc/en/labs/micropython) editor. Follow the instructions next to the download link to install it and open the IDE.
+
+Alternatively, on par with other Arduino products, you can also program your Alvik using Arduino IDE and C++. If this is the case you can find setup instructions over at [Arduino Alvik C++ Tutorial](getting-started.md).
```

Comment:
The `[Arduino Alvik C++ Tutorial](getting-started.md)` link points right back to this "Getting Started with Alvik" page, rather than to a C++ tutorial as expected.

Does a "Arduino Alvik C++ Tutorial" page even exist? Maybe you intended to point the link at the "Setting up Alvik on Arduino IDE" page?

Reply:
Fixed.